src/Goerli-DssSpell.t.sol

```diff
@@ -474,4 +492,77 @@ contract DssSpellTest is GoerliDssSpellTestBase {
+        TeleportOracleAuthLike oracleAuth = TeleportOracleAuthLike(addr.addr("MCD_ORACLE_AUTH_TELEPORT_FW_A"));
+        TeleportRouterLike router = TeleportRouterLike(addr.addr("MCD_ROUTER_TELEPORT_FW_A"));
+
+        bytes32 ilk = "TELEPORT-FW-A";
+        bytes23 domain = "ETH-GOER-A";
+
+        // Sanity checks
+        assertEq(vat.wards(address(join)), 1);
+
+        assertEq(join.wards(address(oracleAuth)), 1);
+        assertEq(join.wards(address(router)), 1);
+        assertEq(join.vow(), address(vow));
+        assertEq(join.daiJoin(), address(daiJoin));
+        assertEq(join.vat(), address(vat));
+        assertEq(join.ilk(), ilk);
+        assertEq(join.domain(), domain);
+
+        assertEq(oracleAuth.signers(0xC4756A9DaE297A046556261Fa3CD922DFC32Db78), 1);
+        assertEq(oracleAuth.signers(0x23ce419DcE1De6b3647Ca2484A25F595132DfBd2), 1);
+        assertEq(oracleAuth.signers(0x774D5AA0EeE4897a9a6e65Cbed845C13Ffbc6d16), 1);
+        assertEq(oracleAuth.signers(0xb41E8d40b7aC4Eb34064E079C8Eca9d7570EBa1d), 1);
+        assertEq(oracleAuth.signers(0x0E0cDcbbE170f6d81f87b45c2227526B6779A083), 1);
+        assertEq(oracleAuth.signers(0x73093A55d5703C7A81D7381F7F24FCf432c64652), 1);
+        assertEq(oracleAuth.signers(0x2a2b83700c990FDFEFD22968fc7C4A4B80783E60), 1);
+        assertEq(oracleAuth.signers(0x1BC7410DD4D18bf8f613F4B6a646FA3953D3A0f2), 1);
+        assertEq(oracleAuth.signers(0xE5D5b00cc04596461a5527616b4F88B754879aE8), 1);
+        assertEq(oracleAuth.signers(0xA5E6053Fe351883036d13C2219b68102AbdFcBB6), 1);
+        assertEq(oracleAuth.signers(0x59524b843866b9686c520fB3d3613A73fe303d30), 1);
+        assertEq(oracleAuth.signers(0x794D810a3d524B9E25227bFA22E69CaaC8544EF2), 1);
+        assertEq(oracleAuth.signers(0xE85963ACc9A361E13306c6395186aa950f750883), 1);
+        assertEq(oracleAuth.signers(0xc65EF2D17B05ADbd8e4968bCB01b325ab799aBd8), 1);
+        
+        assertEq(oracleAuth.teleportJoin(), address(join));
+        assertEq(oracleAuth.threshold(), 1);
+
+        assertEq(router.gateways(domain), address(join));
+        assertEq(router.domains(address(join)), domain);
+        assertEq(router.dai(), address(dai));
+
+        assertEq(CureLike(cure).srcs(CureLike(cure).tCount() - 1), address(join));
+
+        checkTeleportFWIntegration(
+            "OPT-GOER-A",
+            domain,
+            1_000_000 * WAD,
+            addr.addr("OPTIMISM_TELEPORT_BRIDGE"),
+            addr.addr("OPTIMISM_TELEPORT_FEE"),
+            addr.addr("OPTIMISM_ESCROW"),
+            100 * WAD,
+            WAD / 10000,   // 1bps
+            8 days
+        );
+
+        checkTeleportFWIntegration(
+            "ARB-GOER-A",
+            domain,
+            1_000_000 * WAD,
+            addr.addr("ARBITRUM_TELEPORT_BRIDGE"),
+            addr.addr("ARBITRUM_TELEPORT_FEE"),
+            addr.addr("ARBITRUM_ESCROW"),
+            100 * WAD,
+            WAD / 10000,   // 1bps
+            8 days
+        );
+    }
```

Comment:
Might be over-asking, but consider adding a test that cages the vat, and calls end.load(..) while there is teleport debt. Just assuming that we don't have any integration test that checks the cured value on the deployed join.

Reply:
Working on a PR for this one, will post soon and you can decide if it's worth including.

Reply:
https://github.com/makerdao/spells-goerli/pull/103